caikit/core/data_model/base.py

```diff
@@ -666,12 +669,8 @@ def from_proto(cls, proto):
                 )
 
             if field in cls._fields_primitive or field in cls._fields_enum:
-                if field in cls._fields_to_oneof:
-                    if proto.HasField(field):
-                        # "foo_bar_int" has original field name "foo_bar"
-                        original_field_name = "_".join(field.split("_")[:-1])
-                        kwargs[original_field_name] = proto_attr
-                else:
+                # special case for oneofs
```

Comment:
This reverts the changes introduced by https://github.com/caikit/caikit/pull/350/files. We don't need this change